src/main/java/com/ollymonger/dynmap/portals/DynmapPortalsPlugin.java

```
@@ -90,15 +94,15 @@ private void initialiseMarkerApi() {
         this.portalExclusionSet = this.markerApi.getMarkerSet(SET_ID_PORTAL_EXCLUSIONS);
 
         if (this.portalExclusionSet == null) {
-            getLogger().info("Exclusions Set not found, creating new set");
-            this.portalExclusionSet = this.markerApi.createMarkerSet(SET_ID_PORTAL_EXCLUSIONS, SET_NAME_PORTAL_EXCLUSIONS, null, true);
+        getLogger().info("Exclusions Set not found, creating new set");
+        this.portalExclusionSet = this.markerApi.createMarkerSet(SET_ID_PORTAL_EXCLUSIONS, SET_NAME_PORTAL_EXCLUSIONS, null, true);
         }
 
-        this.portalSet.setHideByDefault(false);
+        this.portalSet.setHideByDefault(true);
```

Show portals by default - but hide the exclusion zones